authorization/token/manager/token_manager.go

```diff
@@ -136,14 +137,7 @@ func ContextIdentity(ctx context.Context) (*uuid.UUID, error) {
 	if err != nil {
 		log.Error(ctx, map[string]interface{}{}, "error reading token manager")
 
-		return nil, errors.New("Error reading token manager")
-	}
-	if tm == nil {
```

Comment:
Why removing it? TM still can be null here if it's not in the context. BTW we need add a tests for that.
BTW alternatively maybe we should return/use the default context if it doesn't present in the context..

Reply:
@alexeykazakov AFAIK, if it's not available in the context, the function called above already returns an error (that's what I experienced)

Reply:
Ah, right. Then never mind. Looks good :) Still worth having a test for that ContextIdentity() function but maybe we already have it?